scripts/package-release.sh

```diff
@@ -3,6 +3,12 @@
 set -eo pipefail
 
 RELEASE_DIR="dist/amazon-genomics-cli"
+JQ=$(which jq)
```

Comment:
The `which` command is not found on some linux distros. A more POSIX compliant and portable way to check for the presence of an application is:

```
if ! command -v <the_command> &> /dev/null
then
    echo "<the_command> could not be found"
    exit
fi
```

Reply:
@markjschreiber good point. It's better to stay as close to POSIX as possible.